src/Missings.jl

```diff
@@ -212,21 +212,18 @@ passmissing(f) = PassMissing{Core.Typeof(f)}(f)
 """
    skipmissings(args...)
 
-Returns a tuple of iterators iterating over indices of elements `args`
-where all elements of `args` are not missing.
+Return a tuple of iterators wrapping each of the iterators in `args`, but
+skipping elements at positions where at least one of the iterators returns `missing`.
 
 # Examples
 ```
 julia> x = [1, 2, missing, 4]; y = [1, 2, 3, missing];
 
 julia> tx, ty = skipmissings(x, y)
-(Missings.SkipMissings{Array{Union{Missing, Int64},1}}(Union{Missing, Int64}[1, 2, missing, 
-4], Base.Iterators.Zip{Tuple{Array{Union{Missing, Int64},1},Array{Union{Missing, Int64},1}}}
-((Union{Missing, Int64}[1, 2, missing, 4], Union{Missing, Int64}[1, 2, 3, missing])), 1), 
-Missings.SkipMissings{Array{Union{Missing, Int64},1}}(Union{Missing, Int64}[1, 2, 3, 
-missing], Base.Iterators.Zip{Tuple{Array{Union{Missing, Int64},1},Array{Union{Missing,
- Int64},1}}}((Union{Missing, Int64}[1, 2, missing, 4], Union{Missing, Int64}[1, 2, 3, 
- missing])), 2))
+(SkipMissings{Array{Union{Missing, Int64},1},Tuple{Array{Union{Missing, Int64},1}}}(Union{Mi
```

Comment:
This doesn't seem to match the new `show`, does it?

Reply:
it matches.

Reply:
Where did "comprised of" go then? :-D

Reply:
You are right. The `show` method in this docstring is for the `Tuple` of `(tx, ty)`... Just printing `tx` in the REPL gives the right answer, but the `Tuple` printing overrides it.

Reply:
Ah, indeed. I saw just added a commit to fix this, but actually given that https://github.com/JuliaLang/julia/pull/34734 recommends, 2-argument `show` to print a parseable output, we should probably leave it that way even if it's not so nice.